index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Newton Portfolio Project</title>
+  <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+  <main>
+    <nav>
+      <div class="desktop-menu">
+        <ul>
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+        </ul>
+      </div>
+      <div class="mobile-menu-icon">
+        <img src="./images/Union.svg" alt="">
+      </div>
+    </nav>
+    <section class="home">
+      <div class="introduction">
+        <h2>Hey there.</h2>
+        <h2>I'm Newton.</h2>
+        <h2>I'm a software developer</h2>
```

Comment:
Kindly use an `h1` tag for the main title, "Hey there ..."

* If you want that google (and other search engines) to know about a specific topic, in this case, you, we should use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. 🤗💪

Reply:
@abdulhamiid If I may ask should I change the first h2 to h1 or should I wrap everything under h1

Reply:
Hi @NewtonOmbese kindly wrap everything in an `h1` tag  🤗

Reply:
@abdulhamiid I made this change is this what you meant

![Screenshot from 2023-06-01 08-31-54](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/2d51c4f1-a009-4582-b9ce-0e446ae231cc)